spec/models/contacts_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Contact, type: :model do
+  fixtures :contacts
+  fixtures :users
+
+  context 'name is not empty' do
```

Comment:
The RSpec conventions suggest that context labels start with `when`, `with`, or `without`, e.g `when the name is valid`